src/test/rustdoc/const-display.rs

```diff
@@ -15,7 +15,6 @@
             issue = "0")]
 
 #![feature(rustc_const_unstable, const_fn, foo, foo2)]
-#![feature(min_const_unsafe_fn)]
```

Comment:
Also not sure if this is right wrt. `stage0` or not; please double-check.

Reply:
I don't think this feature gate was ever needed...